networking/sctp/auth/sockopts/runtest.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+# Copyright (c) 2015 Red Hat, Inc. All rights reserved. This copyrighted material 
+# is made available to anyone wishing to use, modify, copy, or
+# redistribute it subject to the terms and conditions of the GNU General
+# Public License v.2.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+#
+# Jianwen Ji: <jiji@redhat.com> 
+
+# include common  and Beaker environments
+. /usr/bin/rhts-environment.sh || exit 1
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+. /mnt/tests/kernel/networking/common/include.sh || exit 1
+
+
+rlJournalStart
+
+########################## Setup ################################
+OUTPUTFILE=$(new_outputfile)
+
+rlPhaseStartSetup
+    rlRun "yum install -y lksctp-tools-devel gcc" 0
```

Comment:
While yum seems to still be aliased to dnf on the new systems it might not be in the future. Can we have the test check if it should use yum or dnf, like https://github.com/CKI-project/tests-beaker/blob/master/distribution/kpkginstall/runtest.sh#L163 ?

Reply:
add checking to use dnf if available in the system